src/main/ApplicationImpl.cpp

```diff
@@ -399,20 +399,26 @@ ApplicationImpl::start()
         throw std::invalid_argument("NODE_IS_VALIDATOR not set");
     }
 
-    if (mConfig.METADATA_OUTPUT_STREAM != "" && mConfig.NODE_IS_VALIDATOR)
+    if (mConfig.METADATA_OUTPUT_STREAM != "" && mConfig.isNetworkedValidator())
     {
         LOG(ERROR) << "Starting stellar-core with METADATA_OUTPUT_STREAM "
-                      "requires NODE_IS_VALIDATOR to be unset";
-        throw std::invalid_argument("NODE_IS_VALIDATOR is set");
+                      "requires NODE_IS_VALIDATOR to be unset or "
```

Comment:
I'm not sure why we log before throwing, I'm pretty sure we log the exception anyways, so logging here seems redundant

Reply:
Good question.  It was like that already; do you want me to change it?

Reply:
Done.